oap-server/server-starter/src/main/resources/ui-initialized-templates/rabbitmq/rabbitmq-node.json

```diff
@@ -1,712 +1,734 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
```

Comment:
No ASF Header? I think this would cause CI failing?

Reply:
The JSON file seems to have been ignored for header license checking.

Reply:
@kezhenxu94 Is this an expected behavior?

Reply:
This is expected, JSON file doesn't support comments at all, and they are intentionally ignore when checking headers, currently all of our UI configuration JSON files have these invalid comments, which I suggested to remove them all in an early PR long time ago which I don't remember now, I had also removed these comment in https://github.com/apache/skywalking/pull/11161/files and I suggest again to remove them in all our JSON files.